libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/plugin/SalesforceNetworkPlugin.java

```diff
@@ -137,16 +148,13 @@ public void onSuccess(RestRequest request, RestResponse response) {
                             // Otherwise return as string
                             callbackContext.success(response.asString());
                         }
+                        // No response
                         else {
                             callbackContext.success();
                         }
                     } catch (Exception e) {
                         SalesforceHybridLogger.e(TAG, "Error while parsing response", e);
-                        if (response.isSuccess()) {
-                            callbackContext.success();
-                        } else {
-                            onError(e);
-                        }
+                        onError(e);
```

Comment:
We couldn't not manipulate the response at all even though it was a successful http.
Should we still return success or fail?